RELEASES.md

```diff
@@ -54,8 +54,8 @@ Stabilized APIs
 ---------------
 
 - [`Cell::update`](https://doc.rust-lang.org/stable/std/cell/struct.Cell.html#method.update)
```

Comment:
```suggestion
- [`Cell::update`](https://doc.rust-lang.org/stable/std/cell/struct.Cell.html#method.update)
- [`mod ffi::c_str`](https://doc.rust-lang.org/stable/std/ffi/c_str/index.html)
```

Reply:
or maybe after the `Default`s? Not sure the best way to sort these...

Reply:
I'll just put it after, IMHO it's more important that it's *in there* rather than a specific perfect ordering.